RELEASENOTES.md

```diff
@@ -122,6 +122,44 @@ first rule
   * ofproto rules that alter FDB learning on OVS are not supported
   * Tagged packets are not supported
 
+### Kubernetes Infrastructure Offload Recipe
+
+The Kubernetes Infrastructure Offload Recipe enhances and extends the
+functionality enabled in 23.01 release. In this release, it adds support for
+Intel ES2K target in addition to existing P4-DPDK target. This is the first
+release that supports both these targets. Following is the list of features
+added in this release.
+
+#### For ES2K target
```

Comment:
Do not use `ES2K target`

We want to use the name `Intel IPU E2100 target` (see Networking Recipe release notes section)

Reply:
I see. We have been using ES2K name in all our K8s documentation. I would put what you have suggested with this name also in parenthesis, so that reader can connect the two terms.